rpm.spec

```diff
@@ -73,6 +38,10 @@ case %cmsplatf in
     export CFLAGS_PLATF="-arch x86_64 -fPIC"
     export LIBS_PLATF="-liconv"
   ;;
+  osx10A_amd64_gcc49*)
```

Comment:
Had I made that change you would have said to make a special case. I can't do anything right.